api/src/tests.rs

```diff
@@ -96,6 +101,7 @@ impl AccessModelStore for MockReconModelTest {
 #[tokio::test]
 #[traced_test]
 async fn create_event() {
+    let _ = init_local_tracing();
```

Comment:
Why do we need this? I thought `traced_test` would init the tracing bits?

Reply:
We probably don't.. I'm not really a fan of `traced_test` so far (or maybe I haven't read the docs). I think I got confused because I wasn't seeing anything because I forgot the `--nocapture` flag, and the `init_local_tracing` just works on RUST_LOG directly without needing that (and it prints nice formatted output).

Reply:
I removed traced_test from the api crate.. happy to go the other way if you have a strong preference.

Reply:
I'd prefer `traced_test` as it should work even without `--nocapture` and uses RUST_LOG directly as well. We use it all over the code base so I'd prefer to keep using it. Can you reproduce the issue you had with it?

Reply:
I put `traced_test` back for now